app/locales/en/translations.js

```diff
@@ -47,6 +47,14 @@ export default {
             refine: `Refine your search by`,
             providers: `Providers`,
             type: 'OSF Registration Type',
+            osf_registraton_filter: 'List of eight OSF registration types',
+            popover: {
+                registries: {
+                    body: "Filtering by Registration type is only available with OSF Registries",
+                    title: "OSF Registration Type Information"
```

Comment:
With the popover changes, can you confirm that "title" is no longer used?

If so, I might suggest removing it to save future translators some work.